content/doc/developer/views/symbols.adoc

```diff
@@ -0,0 +1,56 @@
+---
+title: Symbols
+layout: developerguide
+---
+
+pass:[<span class="version">Available since Jenkins 2.TODO.</span>]
+
+image::/images/developer/views/symbols.svg[Selection of Symbols]
+
+View the complete list of Symbols available to use in Jenkins on link:https://github.com/jenkinsci/jenkins/tree/master/war/src/main/resources/images/symbols[GitHub].
+
+[.lead]
+Jenkins Symbols are an extensive and consistent collection of icons that you can use on Jenkins or in your plugin.
+Symbols are scalable, support different weights and also adapt to the user's theme.
+
+---
+
+=== Using Symbols
+
+Using Symbols in your view is simple. Use the existing `icon` component and set the `src`
+value to the symbol you want, prefixed with `"symbol-"`.
+
+*Jelly example:*
+[source, xml]
+----
+<l:icon src="symbol-search" />
+----
+
+*Groovy example:*
+[source, groovy]
+----
+l.icon(src: 'symbol-search')
+----
+
+As Symbols build on top of the existing `icon` component you can use all the same attributes as

```

Comment:
Documentation should generally be current time, and not referring to things like ‘you could before’, that belongs in upgrade guides, and blogs etc.